api/internal/core/store/query.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
???

Reply:
where is this file come from? please giave the github repo

Reply:
some code from:
https://github.com/kubernetes/dashboard/tree/master/src/app/backend/resource/dataselect
not whole file.